"make istioctl" no longer works

"make istioctl" no longer works.

```
$ make istioctl
make: Nothing to be done for `istioctl'.
```

but the code has changed; I added a typo

```
$ bin/gobuild.sh ${ISTIO_OUT}/istioctl ./istioctl/cmd/istioctl
# istio.io/istio/istioctl/cmd
istioctl/cmd/kubeinject.go:311:40: invalid character U+005C '\'
```

This seems to have been caused by https://github.com/esnible/istio/commit/3ef921b984bcb5f8753c096613c712c4e995c6c2#diff-b67911656ef5d18c4ae36cb6741b7965

I am on OSX.

--- a/Makefile
+++ b/Makefile
@@ -360,7 +360,8 @@ $(ISTIO_OUT)/$(1):
 	STATIC=0 GOOS=$(GOOS) GOARCH=amd64 LDFLAGS=$(3) bin/gobuild.sh $(ISTIO_OUT)/$(1) $(2)
 
 .PHONY: $(1)
-$(1): $(ISTIO_OUT)/$(1)
+$(1):
+	STATIC=0 GOOS=$(GOOS) GOARCH=amd64 LDFLAGS=$(3) bin/gobuild.sh $(ISTIO_OUT)/$(1) $(2)
 
 ifneq ($(ISTIO_OUT),$(ISTIO_OUT_LINUX))
 $(ISTIO_OUT_LINUX)/$(1):

